static/css/admin/scannerrule.css

```diff
@@ -0,0 +1,5 @@
+/* note: this is shared between scannerrule and scannerqueryrule admins */
+
+.model-scannerqueryrule.hide-action-buttons .field-state_with_actions button {
+    visibility: hidden;
```

Comment:
We might want to use `display: block` to save some space in the page.

Reply:
I specifically wanted to keep the same space, so that people like us who might get the view only permission in prod can compare what we see with admins that can edit.

Reply:
Oh ok, I see. Should we have a comment then?

Reply:
e4a14c83a5f8ec9c1e1801ac2ad1ea87099a2ba4